### PR TITLE
Bootstrap redesign

### PR DESCRIPTION
**Pull request by fauxpark**
- Replaced custom CSS and tab functionality with Bootstrap
- Switched from Prism.js to Highlight.js
- Removed vendor JS/CSS and switched to JSDelivr
- Some small tweaks to JS: `document.querySelector()` everywhere, improved copy behaviour
- Updated keycode mappings

Screen Shot 2022-11-05 at 10 55 43

### CODE REVIEW

File: showOutput3.js [hunk 1]
```diff
@@ -311,22 +248,21 @@ const longFormKeycodes = {
...
-    UNICODE_MODE_WIN: 'UC_M_WI',
-    UNICODE_MODE_BSD: 'UC_M_BS',
-    UNICODE_MODE_WINC: 'UC_M_WC'
+    UNICODE_MODE_FORWARD: 'UC_NEXT',
+    UNICODE_MODE_REVERSE: 'UC_PREV',
+    UNICODE_MODE_OSX: 'UC_MAC',
+    UNICODE_MODE_LNX: 'UC_LINX',
+    UNICODE_MODE_WIN: 'UC_WIN',
+    UNICODE_MODE_BSD: 'UC_BSD',
+    UNICODE_MODE_WINC: 'UC_WINC'
```
Comment (noroadsleft) [inline showOutput3.js]: Please revert this block, and re-submit it after the `develop` -> `master` merge. I try to keep the Converter based on `qmk:master`.

**Review by noroadsleft — CHANGES_REQUESTED** [review]
Sorry for the slow response; I didn't see this until yesterday.

Just one change request here and I'll merge it.

File: showOutput3.js (hunk 1 above)
Comment (fauxpark) [reply showOutput3.js]: Done

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**